cli/types/cypress.d.ts

```diff
@@ -2784,6 +2784,13 @@ declare namespace Cypress {
      * @default "cypress/support/{e2e|component}.js"
      */
     supportFile: string | false
+    /**
+     * The test isolation level applied to ensure a clean slate between tests.
+     *   - legacy - resets/clears aliases, intercepts, clock, viewport, cookies, and local storage before each test.
```

Comment:
> resets/clears aliases, intercepts, clock, viewport, cookies, and local storage before each test.

With our reduced scope is all this still true?

Reply:
actually, yes it is, i was thinking you were saying it doesn't reset those things, derp :)

Reply:
It DOES reset those things, but that is the current behavior in Cypress